src/Resources/theme.json

```diff
@@ -237,7 +237,7 @@
                     "de-DE": "Wird über einem Viewport von 991px angezeigt"
                 },
                 "type": "media",
-                "value": "app/storefront/src/assets/logo/logo_header_ver3.gif",
+                "value": "app/storefront/src/assets/logo/logo_header_ver3_desktop.gif",
```

Comment:
what was the reasons of .gif filenames change?

Reply:
After update to shopware 6.1.4 there were issues with duplicated values with `theme.json` file during `theme:dump` command. @bartosz-zasada asked me to push this change.